test/strscan/test_stringscanner.rb

```diff
@@ -207,6 +207,8 @@ def test_pos_unicode
   end
 
   def test_charpos_not_use_string_methods
+    pend "not supported on TruffleRuby" if RUBY_ENGINE == "truffleruby"
```

Comment:
Could you use `omit` instead of `pend`?
`pend` is for a feature that will be implemented later.

Reply:
Done, and it makes the output look better :+1: